### PR TITLE
use language undependant headers in tabular modifications skeletons

### PR DESCRIPTION
**Pull request by dbraquart**
Because country is now a 2-letter code in csv files and in the form, I reverted everything dealing with country translation (cf https://github.com/gridsuite/gridstudy-app/pull/1809)

### CODE REVIEW

**Review by AAJELLAL — APPROVED** [review]
Code OK.
Tests of all equipments OK.

File: src/components/dialogs/network-modifications/tabular-modification/tabular-modification-form.tsx [hunk 1]
```diff
@@ -78,29 +78,37 @@ const TabularModificationForm = () => {
...
+        if (csvTranslatedColumns) {
+            // First comment line contains header translation
+            commentData.push(['#' + csvTranslatedColumns.join(',')]);
+
+            // Optionally a second comment line
+            if (
+                watchType === EQUIPMENT_TYPES.GENERATOR ||
+                watchType === EQUIPMENT_TYPES.LOAD ||
+                watchType === EQUIPMENT_TYPES.SHUNT_COMPENSATOR ||
+                watchType === EQUIPMENT_TYPES.SUBSTATION
```
[the remark this answers is not in the document]
Comment (dbraquart) [reply src/components/dialogs/network-modifications/tabular-modification/tabular-modification-form.tsx:101]: sorry I dont get the benefit of a separate function here (and it should be another Memo in deps or embbeded function).

[the remark this answers is not in the document]
Comment (AAJELLAL) [reply src/components/dialogs/network-modifications/tabular-modification/tabular-modification-form.tsx:101]: > sorry I dont get the benefit of a separate function here (and it should be another Memo in deps or embbeded function).

with a separate function we can manage other equipment types(VL,LINE...) during future developments. This way we centralize these checks in one place but it's not mandatory.
Adding one more deps is not a problem as long as it's well done.

[the remark this answers is not in the document]
Comment (etiennehomer) [reply src/components/dialogs/network-modifications/tabular-modification/tabular-modification-form.tsx:101]: You could do :  
`if (
                !!intl.messages['TabularModificationSkeletonComment/' + watchType]
     )`

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**